tinymce-language-selector/plugin.js

```diff
@@ -74,19 +74,28 @@ tinymce.PluginManager.add('language', function(editor) {
       }
     }
     return selectedLang;
-  }
+  };
 
   const openDialog = (buttonApi) => {
     const selectedNode = editor.selection.getNode();
-    if (['OL', 'UL'].includes(selectedNode.nodeName)) {
+    const selectionStartNode = editor.selection.getStart();
+    const selectionEndNode = editor.selection.getEnd();
+    const listTags = ['OL', 'UL'];
+    const formatTags = ['B', 'U', 'I', 'STRONG', 'EM'];
+    // Inserting a span across multiple tags (excluding formatting) doesn't work.
+    if ((selectionStartNode !== selectionEndNode &&
+        !formatTags.includes(selectionEndNode.nodeName) &&
```

Comment:
Super tiny nano Nit: Switch lines 87 and 88 to check start first. It just reads more logical to me.